### PR TITLE
Fpdf2: spacing , font size 

### PR DESCRIPTION
**Pull request by AmberAnsari89**

### CODE REVIEW

**Comment by AmberAnsari89** [comment]
i created new branch . however it looks it is added to older one

**Comment by AlexWaygood** [comment]
> i created new branch . however it looks it is added to older one

You need to create the new branch from your `main` branch. Maybe you created your new branch from the branch you were using for your previous PR?

**Comment by AmberAnsari89** [comment]
> > i created new branch . however it looks it is added to older one
> 
> You need to create the new branch from your `main` branch. Maybe you created your new branch from the branch you were using for your previous PR?

i think you are right.. i checkout from the previous PR. what i do? shall i create new one?

**Comment by AmberAnsari89** [comment]
Alex,
Are something wrong with this PR?

**Comment by AlexWaygood** [comment]
> Alex, Are something wrong with this PR?

I haven't had a chance to look at it properly yet :)

Please note that we're all volunteers who have day jobs. I'll try to get to it soon.

**Comment by AmberAnsari89** [comment]
i can understand alex, i was mainframe resource and not started python to
learn it more closely. please dont  mind it.

File: stubs/fpdf2/fpdf/graphics_state.pyi [hunk 1]
```diff
@@ -32,25 +32,25 @@ class GraphicsStateMixin:
...
+    def font_size(self) -> float: ...
     @font_size.setter
-    def font_size(self, v) -> None: ...
+    def font_size(self, v: float) -> None: ...
     @property
-    def current_font(self): ...
+    def current_font(self) -> float: ...
     @current_font.setter
-    def current_font(self, v) -> None: ...
+    def current_font(self, v: float) -> None: ...
```
Comment (AlexWaygood) [inline stubs/fpdf2/fpdf/graphics_state.pyi:53]: `current_font` doesn't look like a `float` to me. It's set to a dictionary in the following places:

- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L323
- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L1894

And it's assumed to be a dictionary in the following places:

- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L386
- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L1896
- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L1913
- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L2318

I think `dict[str, Any]` would be a better type hint for `current_font`.

Comment (AmberAnsari89) [reply stubs/fpdf2/fpdf/graphics_state.pyi:53]: you are right..i will do the changes.

**Review by AlexWaygood — APPROVED** [review]
Thanks!

**Pull request merged**